execute/profiler.go

```diff
@@ -128,12 +173,81 @@ func (o *OperatorProfiler) GetResult(q flux.Query, alloc *memory.Allocator) (flu
 			Type:  flux.TString,
 		},
 		{
-			Label: "Begin",
-			Type:  flux.TTime,
+			Label: "Type",
+			Type:  flux.TString,
+		},
+		{
+			Label: "Label",
+			Type:  flux.TString,
+		},
+		{
+			Label: "Count",
+			Type:  flux.TInt,
+		},
+		{
+			Label: "MinDuration",
+			Type:  flux.TInt,
+		},
+		{
+			Label: "MaxDuration",
+			Type:  flux.TInt,
 		},
 		{
-			Label: "End",
-			Type:  flux.TTime,
+			Label: "DurationSum",
+			Type:  flux.TInt,
+		},
+		{
+			Label: "MeanDuration",
+			Type:  flux.TFloat,
+		},
+	}
+	for _, col := range colMeta {
+		if _, err := b.AddCol(col); err != nil {
+			return nil, err
+		}
+	}
+
+	for agg := range o.chOut {
+		b.AppendString(0, "profiler/operator")
+		b.AppendString(1, agg.operationType)
+		b.AppendString(2, agg.label)
+		b.AppendInt(3, agg.resultCount)
+		b.AppendInt(4, agg.resultMin)
+		b.AppendInt(5, agg.resultMax)
+		b.AppendInt(6, agg.resultSum)
+		b.AppendFloat(7, agg.resultMean)
+	}
+	tbl, err := b.Table()
+	if err != nil {
+		return nil, err
+	}
+	return tbl, nil
+}
+
+// GetSortedResult is identical to GetResult, except it accept it calls Sort()
+// on the ColListTableBuilder to make testing easier.
+// sortKeys and desc are passed directly into the Sort() call
+func (o *OperatorProfiler) GetSortedResult(q flux.Query, alloc *memory.Allocator, sortKeys []string, desc bool) (flux.Table, error) {
```

Comment:
I think `GetSortedResult` and `GetResult` can share a common helper function to increase code reusability?